playbooks/frontend.yml

```diff
@@ -8,6 +8,7 @@
     # ansible-playbook -e certbot_test=false frontend.yml
     # - certbot_test: true
     nginx_anonymize_logs: true
+    certbot_domains: api.websearch.org # test!
```

Comment:
Since we have this var in inventory repo in group_vars, I don't want it to be affected so I'm overriding it here

Reply:
Make sense to me. We can remove it before we merge the PR.

However, I suggested before to use `dwebsearch.org` (as it's already set up) and also to create a wildcard certificate (`*.dwebsearch.org`).